### DISCUSSION TITLE
Why youtube don't have lesson 10 which tells ERC20s exist in this repositories?

### DISCUSSION THREAD

**Discussion by WhatMe1on** [Cyfrin/foundry-full-course-cu#2999] [General]
Oh, i got it, the player list on youtebe is sorted with a little confusion.

**Comment by WhatMe1on** [comment]
Lesson 9's later course and Lesson 10  can be find behind the Lesson 11

**Reply by EngrPips** [reply]
Hello @WhatMe1on, I am unsure I understand what the problem is.

**Reply by WhatMe1on** [reply]
I meet the problem that youtube play list seems don't have lesson 10 like this picture
![image](https://github.com/user-attachments/assets/bb8779ca-a371-472f-ade1-2622df2e7b31)

then i scroll down the play list find that lesson 10 are under lesson 11 in youtube play list
![image](https://github.com/user-attachments/assets/77c1a1ef-0183-433d-8688-d74905cf3023)

The problem is already solved, I just use my experience to help others in the same situation

**Reply by EngrPips** [reply]
Oh, I see, Thanks for the heads-up, Friend.
